x-pack/package.json

```diff
@@ -46,27 +47,44 @@
     "@types/d3-time": "^1.0.7",
     "@types/d3-time-format": "^2.1.0",
     "@types/elasticsearch": "^5.0.30",
+    "@types/file-type": "^5.2.1",
+    "@types/get-port": "^3.2.0 ",
```

Comment:
Lets upgrade to `get-port` 4 so we can use the types inside the package. https://www.npmjs.com/package/@types/get-port